.travis.yml

```diff
@@ -4,6 +4,15 @@ python:
   - "2.7"
   - "3.4"
   - "3.5"
+  - "3.6"
+  - "nightly"  # points to 3.7 right now (2016-12-28)
```

Comment:
This isn't something I'm interested in supporting since it may add failures that I'd end up ignoring anyway.